src/DynamoCoreWpf/Views/PackageManager/InstalledPackagesView.xaml

```diff
@@ -60,7 +62,9 @@
 
                     <Border BorderBrush="#222"
                             BorderThickness="0,0,0,1"
-                            Background="#333">
+                            Background="#333"
+                            Width="{Binding Path=ActualWidth, RelativeSource={RelativeSource FindAncestor, 
```

Comment:
can you add a comment here specifying what parent this binding is pointing towards?